### PR TITLE
Feat/set default config

### PR DESCRIPTION
**Pull request by gorexlv**
### Does this pull request fix one issue?

#161 

### Special notes for reviews

### CODE REVIEW

**Comment by louyuting** [comment]
Hi, could you please rebase the newest code?

File: core/config/entity.go [hunk 1]
```diff
@@ -105,3 +105,40 @@ func checkValid(conf *SentinelConfig) error {
 	}
 	return nil
 }
+
+func (entity Entity) AppName() string {
```
Comment (louyuting) [inline core/config/entity.go:109]: Would using a pointer receiver make more sense?

Comment (gorexlv) [reply core/config/entity.go:109]: If the function logic does not modify the Entity,  it does not seem necessary to use pointers.

Comment (louyuting) [reply core/config/entity.go:109]: In future versions, scenarios might exist that read configurations in the slot chain.
This copy operation is not acceptable.

**Pull request merged**